<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela</class>
 <widget class="QWidget" name="tela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Categoria :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data de Movimentação :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Descricão : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cadastro de dados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="caixaCadastrar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="caixaDesc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="caixaCategoria">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Selecione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Despesa Pessoal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Essenciais</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Salario</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="caixaDataMov">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="caixaValor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string> R$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Movimentação Financeira :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="caixaEntSaid">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Selecione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Entrada</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Saida</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
